lib/rubocop/result_cache.rb

```diff
@@ -103,6 +109,7 @@ def save(offenses)
     private
 
     def any_symlink?(path)
+      return if @allow_symlinks_in_cache_location
```

Comment:
I think it's nicer to make this check where the method is called, because saying that there are no symlinks just because symlinks are allowed isn't entirely truthful. So:
```ruby
return if !@allow_symlinks_in_cache_location && any_symlink?(dir)
```